backend/proxy/options.go

```diff
@@ -0,0 +1,23 @@
+package proxy
+
+import "time"
+
+// Options defines options for creating the proxy dialer.
+type Options struct {
+	Timeout   time.Duration
+	KeepAlive time.Duration
+}
+
+// DefaultOptions default timeout/connection options for the proxy.
+var DefaultOptions = Options{
+	Timeout:   180 * time.Second,
+	KeepAlive: 30 * time.Second,
+}
```

Comment:
nit: you could use the `DefaultTimeoutOptions` from the `httpclient` package (there the timeout is 30s, not 180)

Reply:
For sure! Updated